index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, imtial-scale=1.0">
+        <meta http-equiv="X-UA-compatible" content="ie=edge">
+        <link rel="stylesheet" href="styles.css">
+    </head>
+    <body>
+        <main id="main">
+            <h1 id="title">Summer</h1>
+            <div id="img-div">
+            <img id="image" src="image/sun.JPG.jpg"></img>
+            <div id="img-caption"><b>summer is love!</b></div>
+                <div id="tribute-info">Life's better in flip flops.
+                    What time is it? Summertime!
+                    Living for these summer days and summer nights.
+                    Good times and tan lines.
+                    Sunshine is my medicine of choice.
+                    Dog days of summer.
+                    Blue skies, sunshine, everything's fine.
+                    Stay salty.
+                    <a id="tribute-link" href="google.com" target="_blank"></a>
+                    </div>
+                </div>
```

Comment:
- Your code spacing is not aligned correctly:
  - All child elements should be two spaces ahead of their parent element.

**Example:**
```
<div>
  <h1> ... </h1>
  <p>
  .
  .
  </p>
  <div>
    <img ... />
    more elements ...
  </div>
</div>
```